### PR TITLE
OCPBUGS#13156: Updates  4.12+ with additional feedback received

### PR DESCRIPTION
**Pull request by opayne1**
OCPBUGS#13156: Updates 4.12+ with additional feedback received

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-13156

Link to docs preview:
https://60624--docspreview.netlify.app/openshift-enterprise/latest/web_console/dynamic-plugin/overview-dynamic-plugin.html

QE review:
- [x] QE has approved this change.

Additional information:
Changes in this PR include:

- Renaming from plug-in to plugin
- included some of the feedback from the 4.10 PR including the step to install a Helm chart to the example
- other random doc improvements

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm

**Comment by yapei** [comment]
LGTM

File: modules/adding-tab-pods-page.adoc [hunk 1]
```diff
@@ -22,15 +22,16 @@ The {product-title} web console runs in a container connected to the cluster you
...
+. Create a GitHub repository for the template by clicking *Use this template* -> *_Create new repository_*.
 
-. Re-name the new repository with the name of your plugin.
+. Rename the new repository with the name of your plugin.
 
-. Clone your copied repository to your local machine so you can edit the code.
+. Clone the new repository to your local machine so you can edit the code.
 
-. Edit the plugin metadata in the `consolePlugin` declaration of `package.json`.
+. Edit the `package,json` file, adding your plugin's metadata to the `consolePlugin` declaration. For example:
```
Comment (jldohmann) [inline modules/adding-tab-pods-page.adoc:31]:
```suggestion
. Edit the `package.json` file, adding your plugin's metadata to the `consolePlugin` declaration. For example:
```

File: modules/adding-tab-pods-page.adoc [hunk 2]
```diff
@@ -97,5 +98,18 @@ export default function ExampleTab() {
 }
 ----
 
+. Install a Helm chart with the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option to deploy your plugin on a cluster. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
++
+[source,terminal]
+----
+$ helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
```
Comment (jldohmann) [inline modules/adding-tab-pods-page.adoc:106]: this is a suggestion, but I think longer commands benefit from being broken out into multiple lines, e.g. maybe one option or parameter per line for readability:
```suggestion
$ helm upgrade -i  my-plugin charts/openshift-console-plugin \
  -n my-plugin-namespace \
  --create-namespace \
  --set plugin.image=my-plugin-image-location
```

**Review by jldohmann — COMMENTED** [review]
mostly LGTM, left a comment and a suggestion below 🙂

**Pull request merged**